README.rst

```diff
@@ -449,7 +449,7 @@ The following is a basic example of using the **nidmm** module to open a session
         session.configureMeasurementDigits(nidmm.Function.DC_VOLTS, 10, 5.5)
         print("Measurement: " + str(session.read()))
 
-Additional examples for NI-DMM are located in src/nidmm/examples/ directory.
+Additional examples for NI-DMM are located in src/nidmm/examples/ directory `on GitHub <https://github.com/ni/nimi-python/tree/master/src/nidmm/examples>`_.
```

Comment:
This is too verbose in my opinion.
How about you just change it to:

> Other usage examples.

And the whole line is linked.

Reply:
I agree that it could be shorter, though that might be a bit terse. How about:
> Other usage examples can be found on GitHub.

Reply:
Sounds good to me.